icons/asusrog.svg

```diff
@@ -0,0 +1 @@
35417 16.2438 0.839416C15.8827 0.946443 15.5561 1.07331 15.2329 1.20545C15.0421 1.28352 14.4653 1.66689 14.4163 1.76648L14.4162 1.76646V1.76646ZM21.9108 3.6091L22.5919 3.26334C22.8281 3.15152 23.0725 3.02297 23.2949 2.91486C23.7464 2.69553 24.0064 1.9849 23.9999 1.52932C23.9201 1.53266 23.5958 1.68549 23.5077 1.71754C23.4472 1.73956 23.3914 1.75852 23.3347 1.78249C22.9325 1.95238 22.3956 2.15327 22.0143 2.33214C21.9036 2.38409 21.7966 2.42705 21.6845 2.47483C21.5966 2.51233 21.4027 2.58798 21.3666 2.63259C21.2455 2.62648 18.7814 3.84047 18.4743 3.99836C18.037 4.22318 17.4871 4.49263 17.0667 4.72513C17.0169 4.75266 15.7356 5.42689 15.6845 5.49182C15.6002 5.50581 14.7219 6.02436 14.5817 6.103C12.8437 7.07799 10.9188 8.30452 9.2541 9.42702C8.79146 9.73897 8.20054 10.1483 7.73554 10.4966L7.36229 10.7736C7.31938 10.8061 7.26877 10.8277 7.24405 10.8765C7.41002 11.0184 8.50951 11.4688 8.7718 11.5628C10.2439 12.0903 11.6219 12.4579 13.1709 12.7222L14.1538 12.8716L14.9281 12.9396L14.2975 12.8843C14.4179 12.8978 14.5373 12.9173 14.6544 12.9285C14.8276 12.9449 14.9994 12.959 15.1777 12.9767C15.5137 13.0103 15.9038 13.033 16.2491 13.0391C16.6784 13.0466 16.915 13.0684 17.267 12.9206C17.5398 12.806 17.7619 12.6636 17.9787 12.4983C18.7866 11.8828 19.5053 10.9201 20.0788 10.049C20.6788 9.1372 21.2193 8.11852 21.6852 7.10926C21.9224 6.59525 22.1451 6.05894 22.3547 5.51259C22.461 5.23567 22.553 4.96823 22.649 4.67439C22.7022 4.51172 22.8955 3.91785 22.9102 3.78516C22.6899 3.81509 18.8368 5.88191 18.4701 6.07759C17.9729 6.34299 17.4967 6.61944 17.0143 6.87993C16.5455 7.13311 16.0496 7.43228 15.5831 7.69005C14.6352 8.21377 13.6808 8.79715 12.7343 9.34198C12.572 9.4354 11.3667 10.1359 11.3235 10.1851C11.4045 10.1722 15.2256 8.54074 15.3182 8.50274C15.4983 8.42866 19.2501 6.82804 19.3116 6.82538C19.2901 6.91163 19.0579 7.34264 18.9993 7.46933C18.9015 7.68041 18.7838 7.92065 18.6771 8.13058C18.072 9.32046 17.4154 10.7492 16.0945 11.2767L16.0152 11.3239C15.9444 11.3256 15.7509 11.4037 15.6657 11.427C15.5373 11.4621 15.4132 11.4864 15.2838 11.5128C14.675 11.6367 13.7546 11.5721 13.1201 11.4734C12.3966 11.3608 11.89 11.2346 11.1887 11.066C11.0213 11.0258 9.81128 10.6879 9.74143 10.642C9.78082 10.5737 10.4728 10.1328 10.5334 10.091C11.8723 9.1657 14.0965 7.80604 15.5429 6.98788C15.696 6.90127 15.8383 6.82259 15.9782 6.74285C16.0945 6.67654 16.3076 6.56601 16.4015 6.49811L17.0783 6.12907C17.3071 6.00285 17.5173 5.89351 17.7583 5.75675C18.214 5.49824 18.6756 5.28153 19.1242 5.03035C19.1902 4.99342 21.7513 3.6554 21.9108 3.60912V3.6091V3.6091ZM4.50002 10.5934C4.47341 10.5372 4.01093 10.0493 3.91905 9.94731C3.3358 9.29987 2.77039 8.69679 2.18417 8.01658C1.77578 7.54271 1.41198 7.10607 1.0649 6.6874C0.919655 6.51222 0.122625 5.51752 0.000857362 5.3147C-0.0146891 5.3537 0.185964 6.16908 0.215596 6.27622C0.261025 6.44057 0.296163 6.59034 0.342514 6.75805C0.795428 8.39741 1.24803 9.0178 2.80764 9.81014C2.99975 9.90767 4.39633 10.5792 4.50002 10.5934V10.5934Z" fill="black"/></svg>
```

Comment:
extra path attributes like `fille-rule` and `clip-rule` are not allowed

Reply:
I've removed those attributes.

Reply:
I shouldn't have called them extra, these attributes often change the icon in some way and it's not as simple as just removing them. Might have to call another maintainer in because I have had trouble with removing these attributes before.